src/ca65/instr.c

```diff
@@ -303,6 +303,97 @@ static const struct {
     }
 };
 
+/* Instruction table for the 6502 with illegal instructions (X) and DTV 
+** extra opcodes (DTV). Some illegal instructions (X, -DTV?) might be not 
+** supported by DTV. They should be tested on the DTV hardware. 
```

Comment:
Do you have a DTV hardware? Can you conduct the tests?

Reply:
It's fairly accurate in VICE *afaik*. Biggest trap is that some of these do not work 100% the same as on 6502/10 iirc.

Reply:
I have 2 devices (DTV2: 050917 and 050927) however none of them is modded yet sot I cannot upload any programs to them. I think running a illegal instructions test on VICE emulator does not make sense, I do not know how accurate it is. Unfortunately the [related description](https://www.c64-wiki.com/wiki/C64DTV_Programming_Guide#C64_emulation_issues_.28for_the_DTV2.2F3.29) in the programming guide is also rather vague and insufficient. We can remove all illegals codes from my new InsTab6502DTV table in cc65 or they are tagged in comments that they should be verified later on the hardware.